### DISCUSSION TITLE
【任务】9-10月社区激励任务｜40,000 $PEOPLE

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#228] [General]
从8月开始，WTF学院每月的社区激励上调到20,000 $PEOPLE，这包括日常的PR，也包括审核、改善WTF极简教程。这部分奖励优先分配给月度社区激励任务，剩余分给当月贡献者。9-10月社区奖励将一起发放，共计40,000 $PEOPLE。

---
## 任务1：审核WTF Solidity 47-50讲，并补充图片

**赏金**：2,000 $PEOPLE 代币，根据工作量分配，每人最多同时认领2讲。
### 认领情况：
第47讲：可升级合约 **已完成**
@Rulesbreaker 
第48讲：透明代理 **已完成**
@Rulesbreaker 
第49讲：通用可升级代理UUPS
@seasidejuvenile666  **已完成**
第50讲：多签钱包
@seasidejuvenile666 **已完成**

---

## 任务2：审核[WTF Ethers](https://github.com/WTFAcademy/WTF-Ethers) 6-21讲，并补充图片

**赏金**：8,000 $PEOPLE 代币，根据工作量分配，每人最多同时认领4讲。

### 认领情况：

第6讲：部署合约：【[代码](https://github.com/WTFAcademy/WTFEthers/blob/main/06_DeployContract)】 【[教程](https://github.com/WTFAcademy/WTFEthers/blob/main/06_DeployContract/readme.md)】
@seasidejuvenile666 **已完成**
第7讲：检索事件：【[代码](https://github.com/WTFAcademy/WTFEthers/blob/main/07_Event)】 【[教程](https://github.com/WTFAcademy/WTFEthers/blob/main/07_Event/readme.md)】
@seasidejuvenile666 **已完成**
第8讲：合约监听：【[代码](https://github.com/WTFAcademy/WTFEthers/blob/main/08_ContractListener)】 【[教程](https://github.com/WTFAcademy/WTFEthers/blob/main/08_ContractListener/readme.md)】
@seasidejuvenile666 **已完成**
第9讲：事件过滤：【[代码](https://github.com/WTFAcademy/WTFEthers/blob/main/09_EventFilter)】 【[教程](https://github.com/WTFAcademy/WTFEthers/blob/main/09_EventFilter/readme.md)】
@seasidejuvenile666 **已完成**
第10讲：单位转换：【[代码](https://github.com/WTFAcademy/WTFEthers/blob/main/10_Units)】 【[教程](https://github.com/WTFAcademy/WTFEthers/blob/main/10_Units/readme.md)】
@buttonwild **已完成**
第11讲：CallStatic：【[代码](https://github.com/WTFAcademy/WTFEthers/blob/main/11_CallStatic)】 【[教程](https://github.com/WTFAcademy/WTFEthers/blob/main/11_CallStatic/readme.md)】
@buttonwild **已完成**
第12讲：识别ERC721：【[代码](https://github.com/WTFAcademy/WTFEthers/blob/main/12_ERC721Check)】 【[教程](https://github.com/WTFAcademy/WTFEthers/blob/main/12_ERC721Check/readme.md)】
@buttonwild **已完成**
第13讲：编码calldata：【[代码](https://github.com/WTFAcademy/WTFEthers/blob/main/13_EncodeCalldata)】 【[教程](https://github.com/WTFAcademy/WTFEthers/blob/main/13_EncodeCalldata/readme.md)】
@buttonwild **已完成**
第14讲：批量生成钱包：【[代码](https://github.com/WTFAcademy/WTFEthers/blob/main/14_HDwallet)】 【[教程](https://github.com/WTFAcademy/WTFEthers/blob/main/14_HDwallet/readme.md)】
@0xc25fee20 **已完成**

第15讲：批量转账：【[代码](https://github.com/WTFAcademy/WTFEthers/blob/main/15_MultiTransfer)】 【[教程](https://github.com/WTFAcademy/WTFEthers/blob/main/15_MultiTransfer/readme.md)】
@0xc25fee20 **已完成**

第16讲：批量归集：【[代码](https://github.com/WTFAcademy/WTFEthers/blob/main/16_MultiCollect)】 【[教程](https://github.com/WTFAcademy/WTFEthers/blob/main/16_MultiCollect/readme.md)】
@0xc25fee20 **已完成**

第17讲：MerkleTree脚本：【[代码](https://github.com/WTFAcademy/WTFEthers/blob/main/17_MerkleTree)】 【[教程](https://github.com/WTFAcademy/WTFEthers/blob/main/17_MerkleTree/readme.md)】
@beyond009 **已完成**
第18讲：数字签名脚本：【[代码](https://github.com/WTFAcademy/WTFEthers/blob/main/18_Signature)】 【[教程](https://github.com/WTFAcademy/WTFEthers/blob/main/18_Signature/readme.md)】
@beyond009 **已完成**
第19讲：监听Mempool：【[代码](https://github.com/WTFAcademy/WTFEthers/blob/main/19_Mempool)】 【[教程](https://github.com/WTFAcademy/WTFEthers/blob/main/19_Mempool/readme.md)】
@beyond009 **已完成**
第20讲：解码交易数据：【[代码](https://github.com/WTFAcademy/WTFEthers/blob/main/20_DecodeTx)】 【[教程](https://github.com/WTFAcademy/WTFEthers/blob/main/20_DecodeTx/readme.md)】
@beyond009 **已完成**
第21讲：靓号生成器：【[代码](https://github.com/WTFAcademy/WTFEthers/blob/main/21_VanityAddress)】 【[教程](https://github.com/WTFAcademy/WTFEthers/blob/main/21_VanityAddress/readme.md)】
@0xc25fee20 **已完成**

---

## 任务3：整理WTF HTML极简教程2-9讲

**内容**：根据MDN HTML教程，整理WTF HTML极简教程(https://github.com/WTFAcademy/WTF-HTML) 2-9讲。

**要求**：需要学习过HTML。

**赏金**：8,000 $PEOPLE 代币，根据工作量分配，每人最多同时认领2讲。

### 认领情况：

第2讲：HTML元素详解
@seasidejuvenile666 **已完成**
第3讲：常用文本标签
@holazz **已完成**
第4讲：列表
@Hendry5479 **已完成**
第5讲：表单
@Hendry5479 **已完成**
第6讲：超链接
@holazz **已完成**
第7讲：图片
@killerk999 **已完成**
第8讲：语义标签
@killerk999 **已完成**
第9讲：Uniswap页面
@seasidejuvenile666  **已完成**

---

## 任务4：整理WTF CSS极简教程1-20讲

**内容**：根据MDN CSS教程，整理WTF CSS极简教程(https://github.com/WTFAcademy/WTF-CSS) 1-20讲。

**要求**：需要学习过HTML/CSS。

**赏金**：12,000 $PEOPLE 代币，根据工作量分配，每人最多同时认领3讲。

### 认领情况：

**第1讲：Hello CSS**
@AmazingAng **已完成**
**第2讲：CSS语法**（包括嵌入文件的3种方法）
@killerk999 **已完成**
**第3讲：选择器** （样式化HTML元素，改变默认行为，类名，位置，状态）
@killerk999 **已完成**
**第4讲：盒模型** 
@killerk999 **已完成**
**第5讲：背景与边框** 
@Hendry5479 **已完成**
**第6讲：文本与溢出**
@Hendry5479 **已完成**
**第7讲：值，单位，与大小**
@Hendry5479 **已完成**
**第8讲：处理特殊元素** （图像、媒体、表单、表格）
@zhangdiz  **已完成**
**第9讲：布局** （提一下正常布局、弹性、网格、浮动、定位、多栏式）
@holazz **已完成**
**第10讲：弹性布局**
@holazz **已完成**
**第11讲：网格布局**
@holazz **已完成**
**第12讲：定位**
@hubingliang **已完成**
**第13讲：响应式设计** （媒体查询）
@hubingliang  **已完成**
**第14讲：动画与过渡**
@hubingliang  **已完成**
**第15讲：预处理器** （变量）
@zhangdiz  **已完成**
**第16讲：工具与规范** 
@seasidejuvenile666 **已完成**
**第17讲：Uniswap页面**
@seasidejuvenile666 **已完成**
**第18讲：NFT铸造页面** 
@seasidejuvenile666 **已完成**

## 任务5：翻译WTF Solidity极简教程16-30讲

**内容**：根据[Solidity 官方文档](https://docs.soliditylang.org/en/v0.8.17/)的英语标准，翻译[WTF Solidity](https://github.com/AmazingAng/WTF-Solidity)极简教程16-30讲

**要求**：
1. 学习过WTF Solidity教程
2. 参与过WTF Academy社区之前的翻译任务 或 能证明自己的英语能力。
3. 阅读过Solidity官方文档。

**赏金**：15,000 $PEOPLE 代币，根据工作量分配，每人最多同时认领4讲。

### 认领情况：

**第16讲：函数重载**：[代码](https://github.com/AmazingAng/WTFSolidity/blob/main/16_Overloading) | [文章](https://github.com/AmazingAng/WTFSolidity/blob/main/16_Overloading/readme.md) | [Mirror](https://mirror.xyz/ninjak.eth/ADt6g13gwqt42-9_nBxMunSkVHeft6-yb1mp0XH-4f8)
[JustinAsdz](https://github.com/JustinAsdz)
**第17讲：库合约（library）**：[代码](https://github.com/AmazingAng/WTFSolidity/blob/main/17_Library) | [文章](https://github.com/AmazingAng/WTFSolidity/blob/main/17_Library/readme.md) | [Mirror](https://mirror.xyz/ninjak.eth/5nhKjOI40nCT0nCk6iO2TXry3xi09F1Ts4e5lMBlZ1w)
[JustinAsdz](https://github.com/JustinAsdz)
**第18讲：Import**：[代码](https://github.com/AmazingAng/WTFSolidity/blob/main/18_Import) | [文章](https://github.com/AmazingAng/WTFSolidity/blob/main/18_Import/readme.md) | [Mirror](https://mirror.xyz/ninjak.eth/f43II54h19ucfkNDarEdDFGts0lBkJCyrZ57U5Njk3A)
[JustinAsdz](https://github.com/JustinAsdz)
**第19讲：接收ETH（fallback/receive）**：[代码](https://github.com/AmazingAng/WTFSolidity/blob/main/19_Fallback) | [文章](https://github.com/AmazingAng/WTFSolidity/blob/main/19_Fallback/readme.md) | [Mirror](https://mirror.xyz/ninjak.eth/EroVZqHW1lfJFai3umiu4tb9r1ZbDVPOYC-puaZklAw)
[hotsjf](https://github.com/hotsjf)
**第20讲：发送ETH（transfer/send/call）**：[代码](https://github.com/AmazingAng/WTFSolidity/blob/main/20_SendETH) | [文章](https://github.com/AmazingAng/WTFSolidity/blob/main/20_SendETH/readme.md) | [Mirror](https://mirror.xyz/ninjak.eth/NTNHraVAn2OWUKXpr0byphlxl8ytj7fRUAaOaLYfEtA)
[hotsjf](https://github.com/hotsjf)
**第21讲：调用其他合约**：[代码](https://github.com/AmazingAng/WTFSolidity/blob/main/21_CallContract) | [文章](https://github.com/AmazingAng/WTFSolidity/blob/main/21_CallContract/readme.md) | [Mirror](https://mirror.xyz/ninjak.eth/fI48BkuT7wWzRaVKkn1VWxnfLpkUEvIZIJcO5iOIzbk)
[hotsjf](https://github.com/hotsjf)
**第22讲：Call**：[代码](https://github.com/AmazingAng/WTFSolidity/blob/main/22_Call) | [文章](https://github.com/AmazingAng/WTFSolidity/blob/main/22_Call/readme.md) | [Mirror](https://mirror.xyz/ninjak.eth/vMJ0AHl3b_A5Lyey5lLdFhzkGT9wJ-D6vJpZpH0vlls)
[hotsjf](https://github.com/hotsjf)
**第23讲：Delegatecall**：[代码](https://github.com/AmazingAng/WTFSolidity/blob/main/23_Delegatecall) | [文章](https://github.com/AmazingAng/WTFSolidity/blob/main/23_Delegatecall/readme.md) | [Mirror](https://mirror.xyz/ninjak.eth/hW9iLCOUUqsRCCZbzGS3x5m0wXfJBhlYQWLq93rplcU)
[shao-shuai](https://github.com/shao-shuai)
**第24讲：在合约中创建新合约**：[代码](https://github.com/AmazingAng/WTFSolidity/blob/main/24_Create) | [文章](https://github.com/AmazingAng/WTFSolidity/blob/main/24_Create/readme.md) | [Mirror](https://mirror.xyz/ninjak.eth/kojopp2CgDK3ehHxXc_2fkZe87uM0O5OmsEU6y83eJs)
[shao-shuai](https://github.com/shao-shuai)
**第25讲：Create2**：[代码](https://github.com/AmazingAng/WTFSolidity/blob/main/25_Create2) | [文章](https://github.com/AmazingAng/WTFSolidity/blob/main/25_Create2/readme.md) | [Mirror](https://mirror.xyz/ninjak.eth/Dz7kd6-RrcNvdHnptWKzsW9AcIOSA8PoVsMsQh8nlEM)
[seasidejuvenile666](https://github.com/seasidejuvenile666)
**第26讲：删除合约**：[代码](https://github.com/AmazingAng/WTFSolidity/blob/main/26_DeleteContract) | [文章](https://github.com/AmazingAng/WTFSolidity/blob/main/26_DeleteContract/readme.md) | [Mirror](https://mirror.xyz/ninjak.eth/080JeNq8X_HfciftWa-ed0S6Wb5YFuIb5GPVISu1gIc)
[Hendry5479](https://github.com/Hendry5479)
**第27讲：ABI编码解码**：[代码](https://github.com/AmazingAng/WTFSolidity/blob/main/27_ABIEncode) | [文章](https://github.com/AmazingAng/WTFSolidity/blob/main/27_ABIEncode/readme.md) | [Mirror](https://mirror.xyz/ninjak.eth/jXJnvwkoQzvJaqVIxagxneSZim6Qxm-StuNNxLuKuw8)
[seasidejuvenile666](https://github.com/seasidejuvenile666)
**第28讲：Hash**：[代码](https://github.com/AmazingAng/WTFSolidity/blob/main/28_Hash) | [文章](https://github.com/AmazingAng/WTFSolidity/blob/main/28_Hash/readme.md) | [Mirror](https://mirror.xyz/ninjak.eth/csiRSVEpEwj7f0OOtSpm1KdGHRqD4fNogmiahHx60nE)
[Hendry5479](https://github.com/Hendry5479)
**第29讲：函数选择器（selector）**：[代码](https://github.com/AmazingAng/WTFSolidity/blob/main/29_Selector) | [文章](https://github.com/AmazingAng/WTFSolidity/blob/main/29_Selector/readme.md) | [Mirror](https://mirror.xyz/ninjak.eth/_Q-N_VGUV8F4QZbggR8Swv16LStBdfkeQb8qwSfoNTw)
[Hendry5479](https://github.com/Hendry5479)
**第30讲：Try-Catch**：[代码](https://github.com/AmazingAng/WTFSolidity/blob/main/30_TryCatch) | [文章](https://github.com/AmazingAng/WTFSolidity/blob/main/30_TryCatch/readme.md) | [Mirror](https://mirror.xyz/ninjak.eth/asj1-eKf6YdIoI9oMSLI3VECwqCzRHwaqIS01ALQ7qo)
[seasidejuvenile666](https://github.com/seasidejuvenile666)
## 认领方式
在这个帖子下面回复就能认领，认领人需要填写过[WTF社区成员申请表格](https://docs.google.com/forms/d/e/1FAIpQLSe4KGT8Sh6sJ7hedQRuIYirOoZK_85miz3dw7vA1-YjodgJ-A/viewform?usp=sf_link)。

有用的资料：
1. Markdown书写规则（**必看**）：https://www.markdownguide.org/cheat-sheet/
4. 审核并添加图片后的例子：https://github.com/AmazingAng/WTFSolidity/tree/main/02_ValueTypes
5. 审核第一期：https://github.com/AmazingAng/WTFSolidity/discussions/18
6. https://github.com/AmazingAng/WTFSolidity/discussions/132
7. 8月社区激励任务：https://github.com/AmazingAng/WTF-Solidity/discussions/174

**Comment by 0xc25fee20** [comment]
任务二，14、15、16、21讲

**Reply by AmazingAng** [reply]
好的

**Comment by Rulesbreaker** [comment]
任务一：47、48讲

**Reply by AmazingAng** [reply]
ok

**Comment by Ayzl999** [comment]
任务三: 7、8

**Reply by AmazingAng** [reply]
收到

**Comment by Hendry5479** [comment]
任务三:  4、5讲

**Reply by AmazingAng** [reply]
收到

**Comment by holazz** [comment]
任务三: 3、6讲

**Reply by AmazingAng** [reply]
收到

**Comment by seasidejuvenile666** [comment]
认领
任务一：49 50讲
任务二：6 7 8 9讲
任务三：3、9讲

**Reply by seasidejuvenile666** [reply]
任务三认领2 9讲 上面打错了

**Reply by AmazingAng** [reply]
收到

**Comment by beyond009** [comment]
任务二17 18 19 20

**Reply by AmazingAng** [reply]
ok

**Comment by buttonwild** [comment]
任务二10 11 12 13

**Reply by AmazingAng** [reply]
ok

**Comment by lcy101u** [comment]
任務二 12 13

**Reply by AmazingAng** [reply]
目前任务已经被领完了，要是认领者没有按时完成的话，会通知你，感谢！

**Comment by shaobaili3** [comment]
任务二14 15 16

**Reply by AmazingAng** [reply]
目前任务已经被领完了，要是认领者没有按时完成的话，会通知你，感谢！

**Comment by holazz** [comment]
任务四 9-11

**Reply by AmazingAng** [reply]
ok

**Comment by Ayzl999** [comment]
任务4 2-4

**Reply by AmazingAng** [reply]
好的

**Comment by Hendry5479** [comment]
任务4 5 - 7

**Reply by AmazingAng** [reply]
ok

**Comment by seasidejuvenile666** [comment]
任务4的16 17 18 给我吧

**Reply by AmazingAng** [reply]
好的

**Comment by hubingliang** [comment]
css 第14讲 第12讲

**Reply by AmazingAng** [reply]
现在还剩 8,12,13,14,15

**Reply by hubingliang** [reply]
> 现在还剩 8,12,13,14,15

12-14

**Reply by AmazingAng** [reply]
好的

**Comment by zhangdiz** [comment]
任务四 的8 15

**Reply by AmazingAng** [reply]
现在还剩 8,15

**Comment by shao-shuai** [comment]
请问还有未领取的任务吗？俺也想做一个

**Comment by JustinAsdz** [comment]
认领任务5：16，17，18

**Reply by AmazingAng** [reply]
收到

**Comment by hotsjf** [comment]
19-22

**Reply by AmazingAng** [reply]
收到

**Comment by shao-shuai** [comment]
认领任务5: 23，24

**Reply by AmazingAng** [reply]
收到

**Comment by Hendry5479** [comment]
认领任务5: 26，28，29

**Reply by AmazingAng** [reply]
收到

**Comment by AmazingAng** [comment]
25,27,30 先留给 @seasidejuvenile666

**Reply by seasidejuvenile666** [reply]
ok

**Comment by MarkTangCd** [comment]
期待后续任务
